.circleci/config.yml

```diff
@@ -10,6 +10,14 @@ references:
         `which python3` -m venv venv
         venv/bin/pip install -U setuptools pip wheel
         venv/bin/pip install -r requirements/base.txt
+        venv/bin/pip install -U coverage
+
+  create-tmp-dir: &create-tmp-dir
+    run:
+      name: Create workspace temporary directories
+      command: |
+        mkdir -p tmp/
+        mkdir -p tmp/coverage/
```

Comment:
Couldn't you use a single command, `mkdir -p tmp/coverage/`, to create the directory structure

Reply:
Ah, yes. I noticed that when I C&P'd it from the CCCD config and forgot to come back to it 🤦